packages/eslint-config-eslint/index.js

```diff
@@ -13,17 +14,28 @@ const unicorn = require("eslint-plugin-unicorn");
 jsdoc.configs.recommended.plugins = { jsdoc };
 eslintComments.configs.recommended.plugins = { "eslint-comments": eslintComments };
 
-// extends eslint-plugin-n's recommended config
-const nodeConfigs = [nodeRecommendedConfig, {
+// extends eslint-plugin-n's config
+const nodeSharedRules = {
+    "n/callback-return": ["error", ["cb", "callback", "next"]],
+    "n/handle-callback-err": ["error", "err"]
+};
+const nodeCommonJSConfig = {
+    ...nodeRecommendedScriptConfig,
     rules: {
-        "n/callback-return": ["error", ["cb", "callback", "next"]],
-        "n/handle-callback-err": ["error", "err"],
-        "n/no-deprecated-api": "error",
```

Comment:
I removed `n/no-deprecated-api` from here because I believe it's already in eslint-plugin-n's recommended configs. Is that correct?

Reply:
👍